cli/src/tasks/init.ts

```diff
@@ -166,10 +166,10 @@ async function mergeConfig(
 function printNextSteps(newConfigName: string) {
   logSuccess(`${c.strong(newConfigName)} created!`);
   output.write(
-    `\nAdd platforms using ${c.input('npx cap add')}:\n` +
-      `  ${c.input('npx cap add android')}\n` +
-      `  ${c.input('npx cap add ios')}\n\n` +
-      `Follow the Developer Workflow guide to get building:\n${c.strong(
+    `\nCheck next steps: \n${c.strong(
+      `https://capacitorjs.com/docs/v3/getting-started#where-to-go-next`,
```

Comment:
I wonder if we should just link to the iOS and Android platform guides directly.

Reply:
the "where to go next" has links to the iOS guide, the Android guide and the workflow, so didn't want to add that many links. In fact, I could probably remove the workflow link since it's included in the "where to go next"

Reply:
👍